map_model/src/map.rs

```diff
@@ -191,8 +185,12 @@ impl Map {
         &self.intersections
     }
 
-    pub fn all_turns(&self) -> &BTreeMap<TurnID, Turn> {
-        &self.turns
+    pub fn all_turns(&self) -> Vec<&Turn> {
```

Comment:
This seems likely to be substantially more expensive than before. 


What about something like:

```
pub fn all_turns(&self) -> impl Iterator<Item = Turn> {
    self.intersections.iter().flat_map(|i| i.turns.iter())
}
```

Not sure if this will work in all our use cases...

Reply:
Thanks, done! I always forget that Rust makes it pretty easy to lazily evaluate